[Executables] run_tests3.bat + disable __pycache__

gpotter2 opened the pull request.

p-l- commented:
Could you use names similar to the unix ones ("xxx.bat", "xxx_py2.bat", "xxx_py3.bat")?

gpotter2 commented:
@p-l- Updated

gpotter2 commented on test/run_tests_py3:
```diff
@@ -1,3 +1,4 @@
 #! /bin/sh
 PYTHON=python3
+PYTHONDONTWRITEBYTECODE="True" # Disable __pycache__
```
Ok

gpotter2 commented:
So should I add a run_scapy.bat.lnk file ? (That's a shortcut file, as symlinks are unusable on windows)

p-l- commented:
@gpotter2 maybe a simple copy? BTW what happens to symlinks when you clone a repository (e.g., `README -> README.md` in Scapy's root)?

gpotter2 commented:
`README` is not-openable file containing a single line of text:
`README.md`

gpotter2 commented:
Update:
- fixed PYTHONDONTWRITEBYTECODE
- added run_scapy.bat, which selects either python 2 or python 3 if available (python 2 by default at least for now)
- updated run_scapy_py2 and run_scapy_py3 with a `--nopause` optional parameter, to manually force-disable the final PAUSE (which triggered only when scapy crashes)
